src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.form1 {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.firstSearch {
+  width: 100%;
+  height: 70px;
+  /* margin-left: 10px;
+  margin-right: 10px; */
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.firstSearch::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.firstSearch:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.firstSearch:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  outline: none;
+  font-family: AvenirHeavy, sans-serif;
+}
+
+.firstIcon {
+  position: absolute;
+  top: 45px;
+  left: 31px;
+}
```

Comment:
where is the relative element of this icon?
add relative position to label and change position of the image.
fore hints can use this [link](https://css-tricks.com/absolute-positioning-inside-relative-positioning/)